client/informant/src/display.rs

```diff
@@ -66,15 +69,29 @@ impl<B: BlockT> InformantDisplay<B> {
 			(SyncState::Downloading, Some(n)) => (format!("Syncing{}", speed), format!(", target=#{}", n)),
 		};
 
+		// Colour informant output if we're running natively.
+
+		#[cfg(not(target_os = "unknown"))]
```

Comment:
As usual, platform-specific code really isn't a great thing.
I'd prefer to have a runtime flag that allows enabling/disabling colour output.

Reply:
On the one hand, I agree.  On the other hand, what if the flag is enabled when compiling for wasm?  That cannot possibly work.  So we need to give an error here.

Reply:
> That cannot possibly work.

Adding colours is just about inserting control characters within the string. If the flag is enabled on WASM, everything will still work perfectly.

Reply:
I've added a `Configuration` option for this, but not a cli flag.